x-pack/metricbeat/module/azure/monitor/_meta/fields.yml

```diff
@@ -0,0 +1,39 @@
+- name: monitor
+  type: group
+  description: >
+    monitor
+  fields:
+    - name: resource
+      type: group
+      description: >
+        The resource specified
+      fields:
+       - name: name
+         type: keyword
+         description: >
+           The name of the resource
+       - name: type
+         type: keyword
+         description: >
+           The type of the resource
+    - name: namespace
+      type: keyword
+      description: >
+        The namespace selected
+    - name: subscriptionID
+      type: keyword
+      description: >
+        The subscription ID
+    - name: metrics.*.*
```

Comment:
I thought they are are in cloudwatch https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/aws/cloudwatch/_meta/fields.yml. Correct me if I am wrong.

Reply:
It will be prefixed by `aws.cloudwatch.*`. fields.yml fields are concatenated together. Best double check in the resulting template to confirm.

Reply:
> I thought they are inside cloudwatch https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/aws/cloudwatch/_meta/fields.yml. Correct me if I am wrong.

Umm, in https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/aws/cloudwatch/_meta/data.json they are in `aws.metrics.*`, @kaiyan-sheng could you confirm?

Reply:
@jsoriano Sorry I missed this comment. `cloudwatch` metricset stores data in `aws.metrics.*` now. Because with using light module to create other metricsets, we don't want `aws.cloudwatch` being the prefix for other services, such as elb or ebs.

Reply:
Thanks, I think we should use the same pattern here for consistency. @narph what do you think? It'd mean to store azure metrics in `azure.metrics.*` instead of `azure.monitor.metrics.*`.
I think that other common fields like the namespace or the subscription ID could be placed also at the module level.